include/llbuild/Basic/PlatformUtility.h

```diff
@@ -0,0 +1,30 @@
+//===- PlatformUtility.h ----------------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2015 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file implements small platform compatability wrapper functions for
+// common functions.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLBUILD_BASIC_PLATFORMUTILITY_H
+#define LLBUILD_BASIC_PLATFORMUTILITY_H
+
+namespace llbuild {
+namespace basic {
+class PlatformUtility {
```

Comment:
I'd just make this a free function, and if you really want it to be in a namespace add another one inside llbuild::basic (e.g., `llbuild::basic::sys` would make sense to me).